src/aotextarea.cpp

```diff
@@ -32,9 +32,8 @@ void AOTextArea::append_chatmessage(QString p_name, QString p_message,
     p_message += " ";
   }
 
-  QString result = p_message.toHtmlEscaped()
-                       .replace("\n", "<br>")
-                       .replace(url_parser_regex, "<a href='\\1'>\\1</a>");
+  QString result = this->closetags(p_message);
+  result = result.replace("\n", "<br>").replace(url_parser_regex, "<a href='\\1'>\\1</a>");
```

Comment:
For url parse regex we'll need to *not* do that if it's already handled via html